tutorials/create-events-dataset.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: page
+title: Creating the Events Dataset
+---
+
+This lesson shows you how to create a dataset suitable for storing standard event records. You define a dataset schema, a partition strategy, and a URI that specifies the storage scheme.
```

Comment:
The next section goes into detail about what a standard event is. If the consumer is momentarily intrigued and wonders what a standard event could be, their suspense will be short-lived.